### PR TITLE
Replaced Examples Folder label for How Tos

### PR DESCRIPTION
**Pull request by lucas-tortora**
# Description of change

* Replaced Examples Folder label for How Tos
* Added links

## Type of change

- Enhancement (a non-breaking change which adds functionality)
- Documentation Fix

## How the change has been tested

Wiki was built locally

## Change checklist

- [x] I have followed the contribution guidelines for this project
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by lucas-tortora** [comment]
@Dr-Electron  could you take a look whenever you have a minute?

File: documentation/docs/getting_started/README.md [hunk 1]
```diff
@@ -35,8 +35,8 @@ official installation instructions. For example:
 
 You should use this subsection to give the user concrete and concise instructions on how to install the project.
 Ideally, these should be very short, and if an in depth explanation is required to install extensions or
-non-essential software you should address it in the [examples](../examples/README.mdx) or
-[tutorials](../tutorials/README.mdx) folders.
+non-essential software you should address it in the [How Tos](../how-tos/README.mdx) or
```
Comment (Dr-Electron) [inline documentation/docs/getting_started/README.md]: To be consistent can we use how_tos?
```suggestion
non-essential software you should address it in the [How Tos](../how_tos/README.mdx) or
```

File: documentation/docs/getting_started/README.md [hunk 2]
```diff
@@ -55,7 +55,7 @@ yarn add @iota/client-wasm
 You should use this subsection to provide the users with the simplest possible example in which they can use the project
 after installing it, as well as minimal explanations if required. Keep in mind that as in the
 [Install the Project](#install-the-project) subsection, complex use cases should be addressed in the
-[examples](../examples/README.mdx) or [tutorials](../tutorials/README.mdx). For example:
+[How Tos](../how-tos/README.mdx) or [Tutorials](../tutorials/README.mdx). For example:
```
Comment (Dr-Electron) [inline documentation/docs/getting_started/README.md]:
```suggestion
[How Tos](../how_tos/README.mdx) or [Tutorials](../tutorials/README.mdx). For example:
```

File: documentation/docs/key_concepts/README.md [hunk 3]
```diff
@@ -22,7 +22,7 @@ different subject while doing so, you should add a link to that subjects explana
 The main topic should be the first header in the article, and any subtopics (or even a subtopic's subtopic) should
 follow hierarchically by using the appropriate header tag level.
 
-## Related Examples
+## Related How Tos And Tutorials
 
-Your explanation articles will probably relate to one or more examples or tutorials. Please make sure to list them
+Your explanation articles will probably relate to one or more [How Tos](../how-tos/README.mdx) or [tutorials](../tutorials/README.mdx). Please make sure to list them
```
Comment (Dr-Electron) [inline documentation/docs/key_concepts/README.md]:
```suggestion
Your explanation articles will probably relate to one or more [How Tos](../how_tos/README.mdx) or [tutorials](../tutorials/README.mdx). Please make sure to list them
```

File: documentation/docs/tutorials/README.mdx [hunk 4]
```diff
@@ -9,6 +9,6 @@ keywords:
 
 A [tutorial](https://diataxis.fr/tutorials/) is the most complex type of documentation you will have. They usually
 require regular effort to maintain them, touch a variety of subjects and should guide the user in much more detail than
-[how-tos](../examples/README.mdx) or [explanations](../key_concepts/README.md).
+[How Tos](../how-tos/README.mdx) or [Key Concepts](../key_concepts/README.md).
```
Comment (Dr-Electron) [inline documentation/docs/tutorials/README.mdx]:
```suggestion
[How Tos](../how_tos/README.mdx) or [Key Concepts](../key_concepts/README.md).
```

File: documentation/sidebars.js [hunk 5]
```diff
@@ -23,11 +23,11 @@ module.exports = {
     },
     {
       type: "category",
-      label: "Examples",
+      label: "How Tos",
       items: [
         {
           type: "doc",
-          id: "examples/README",
+          id: "how-tos/README",
```
Comment (Dr-Electron) [inline documentation/sidebars.js]:
```suggestion
          id: "how_tos/README",
```

**Review by Dr-Electron — COMMENTED** [review]
LGTM. Only I would rename the folder to ho_tos as we use underscore everywhere else too?Wdyt?

**Comment by lucas-tortora** [comment]
> LGTM. Only I would rename the folder to ho_tos as we use underscore everywhere else too?Wdyt?

sounds good to me, updated :)

**Review by Dr-Electron — APPROVED** [review]
LGTM

**Pull request merged**